product_kosher/models/product_kosher.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 OpenSynergy Indonesia
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields, api
+from openerp.tools.translate import _
+from openerp.exceptions import Warning as UserError
+
+
+class ProductKosher(models.Model):
+    _name = "product.kosher"
+    _description = "Product Kosher"
+    _order = "date_start desc"
+
+    name = fields.Char(
+        string="# Certificate",
+        required=True,
+    )
+    issuer_id = fields.Many2one(
+        string="Issuer",
+        comodel_name="res.partner",
+        )
+    product_tmpl_id = fields.Many2one(
+        string="Product Template",
```

Comment:
Forgive if this is obvious (I don't deal with Kosher stuff), but wouldn't the cert be at the variant level instead? Wouldn't all variations need to be individually certified?